frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -39,6 +39,7 @@
         vm.columnIndexSort = 0;
         // save initial ranking
         vm.initial_ranking = {};
+        vm.status = "";
```

Comment:
What is the use of this variable? Am I missing something?

Reply:
@RishabhJain2018  I am using this variable in `ng-if` to render `edit startdate/enddate` form.